packages/babel-parser/src/parse-error/standard-errors.ts

```diff
@@ -35,6 +35,7 @@ export default {
   AwaitNotInAsyncContext:
     "'await' is only allowed within async functions and at the top levels of modules.",
   AwaitNotInAsyncFunction: "'await' is only allowed within async functions.",
+  // TODO: Accesor -> Accessor
   BadGetterArity: "A 'get' accesor must not have any formal parameters.",
   BadSetterArity: "A 'set' accesor must have exactly one formal parameter.",
```

Comment:
This is kind of annoying, delaying to babel8 means we have to copy and paste a lot of tests, and modify it immediately I'm not sure if it will affect compatibility.

Reply:
We can do it in a separate PR. 

We don't have to defer to Babel 8: The error message improvement can be shipped in patch releases, since it is not part of our API protocol. This approach is also similar to how Node.js treats its error message.